src/main.cpp

```diff
@@ -313,7 +313,7 @@ int main(int argc, char **argv) {
    PlayerRenderComponent* playerRenderComp = new PlayerRenderComponent(shapeGirl,
       "Phong", pearl);
    GameObject* player = new GameObject(
-      GameObjectType::DYNAMIC_OBJECT,
+      GameObjectType::PLAYER,

```

Comment:
that screwed me over so bad, it took me a while to figure out why the cookies would immediately collide with the player itself xD